.github/workflows/rust.yml

```diff
@@ -71,6 +71,32 @@ jobs:
           # we run tests using BitVec<u64,_> which doesn't.
           args: --all-features --target wasm32-unknown-unknown
 
+  no_std:
+    name: Check no_std build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install Rust stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: aarch64-unknown-none
+          override: true
+
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v1.3.0
+
+      - name: Check no_std build
+        uses: actions-rs/cargo@v1.0.3
+        with:
+          command: check
+          # The aarch64-unknown-none doesn't support `std`, so this
+          # will fail if the crate is not no_std compatible.
+          args: --no-default-features --target aarch64-unknown-none --features bits,primitive-types,derive
```

Comment:
IIRC, `--target aarch64-unknown-none` is used here because `--feature std` is leaked within workspace?

Otherwise, I don't understand why `--target aarch64-unknown-none` is used here....

Reply:
Yes, exactly. In my experience with `no_std` is that while `#![no_std]` ensures that std is not used in the current workspace, it does not guarantee that dependencies are `no_std` as well.